src/Platforms/SQLServerPlatform.php

```diff
@@ -1777,11 +1778,17 @@ private function generateIdentifierName($identifier): string
 
     protected function getCommentOnTableSQL(string $tableName, ?string $comment): string
     {
+        if (str_contains($tableName, '.')) {
+            [$schemaName, $tableName] = explode('.', $tableName);
+        } else {
+            $schemaName = 'dbo';
```

Comment:
We still have to default to the `dbo` schema here because the platform doesn't know of the current schema, which is scoped to a connection.